### PR TITLE
Add Mechanical Ink podcast

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Related Issue(s)
_Reminder to add related issue(s), if available._

- https://github.com/oddbird/oddFlock/issues/116

### CODE REVIEW

File: content/talks/containers.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ tags:
   - Container Queries
   - CSSWG
 events:
+  - venue: CMD Amsterdam
+    date: 2024-06-05
+    adr: Amsterdam, Netherlands
+    slides: https://slides.oddbird.net/cq/pre-cssday/
```
Comment (jgerigmeyer) [inline content/talks/containers.md:20]: Do we have a URL to add here?

**Review by jgerigmeyer — APPROVED** [review]

**Review by SondraE — APPROVED** [review]
The first paragraph is in present tense. I think you could change the second paragraph below the video to: "Our discussion span_s_..."

Our style guide says that for headlines we should, _Use Title Case: Capitalize the first letter of every word except articles, prepositions, and conjunctions._, but maybe you're doing it with lowercase to match the way Mechanical Ink did it?

File: content/talks/containers.md (hunk 1 above)
Comment (mirisuzanne) [reply content/talks/containers.md:20]: We could add the eventbrite link: https://www.eventbrite.nl/e/pre-css-day-tickets-872463962197

**Comment by mirisuzanne** [comment]
> … but maybe you're doing it with lowercase to match the way Mechanical Ink did it?

@SondraE well, kinda: I did a copy, and then a paste -- and didn't bother to change it

File: content/blog/elsewhere/interview/podcast/mechanical-ink.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: Collaboration, the future of CSS, Sass, and the web with Miriam Suzanne
```
Comment (mirisuzanne) [inline content/blog/elsewhere/interview/podcast/mechanical-ink.md:2]:
```suggestion
title: Collaboration, the Future of CSS, Sass, and The Web with Miriam Suzanne
```

File: content/talks/containers.md (hunk 1 above)
Comment (mirisuzanne) [reply content/talks/containers.md:20]:
```suggestion
    url: https://www.eventbrite.nl/e/pre-css-day-tickets-872463962197
    slides: https://slides.oddbird.net/cq/pre-cssday/
```

File: content/blog/elsewhere/interview/podcast/mechanical-ink.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    span: full
+---
+
+{% import "embed.macros.njk" as embed %}
+{{ embed.figure(
+  data=media,
+  caption="Video teaser, and full audio of our conversation"
+) }}
+
+Our discussion spanned a range of topics,
```
Comment (mirisuzanne) [inline content/blog/elsewhere/interview/podcast/mechanical-ink.md:34]:
```suggestion
Our discussion spans a range of topics,
```

File: content/blog/elsewhere/interview/podcast/mechanical-ink.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+title: Collaboration, the Future of CSS, Sass, and The Web with Miriam Suzanne
```
Comment (mirisuzanne) [inline content/blog/elsewhere/interview/podcast/mechanical-ink.md:2]:
```suggestion
title: Collaboration, the Future of CSS, Sass, and the Web with Miriam Suzanne
```

**Pull request merged**